knoten/csm.py

```diff
@@ -245,7 +252,7 @@ def generate_latlon_boundary(camera, boundary, dem=0.0, radii=None, **kwargs):
 
         gnds[i] = [gnd.x, gnd.y, gnd.z]
 
-    lons, lats, alts = pyproj.transform(ecef, lla, gnds[:,0], gnds[:,1], gnds[:,2])
+    lons, lats, alts = transformer.transform(ecef, lla, gnds[:,0], gnds[:,1], gnds[:,2])
```

Comment:
Where do ecef and lla come from?

Reply:
Good catch. Those need to be removed. Commit incoming.